examples/spark/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+import java.util.Properties
+
+plugins {
+    kotlin("jvm") version "1.3.70"
+    application
+}
+
+allprojects {
```

Comment:
no need for those repos as it should be defined in the parent example gradle build file